kubectl-cloudflow/domain/cloudflowapplication.go

```diff
@@ -173,6 +183,12 @@ type CloudflowApplicationDescriptorDigestPair struct {
 	ImageDigest   string `json:"imageDigest"`
 }
 
+// CloudflowStreamletDescriptorsDigestPair is a pair of streamlet descriptors and image digest
+type CloudflowStreamletDescriptorsDigestPair struct {
+	StreamletDescriptors []Descriptor `json:"streamletDescriptors"`
+	ImageDigest          string       `json:"imageDigest"`
```

Comment:
What is the digest?

Reply:
ah this is the 'sha'  for the image, right?

Reply:
yes ..